docs/releasenotes/3.0.0.rst

```diff
@@ -1,9 +1,28 @@
-
 3.0.0
-=====
+-----
+
+Deprecations
+============
```

Comment:
Removals are "Backwards Incompatible Changes", "Deprecations" is for when warnings are added in earlier releases.